app/controllers/api/v1/web_hooks_controller.rb

```diff
@@ -6,8 +6,8 @@ class Api::V1::WebHooksController < Api::BaseController
 
   def index
     respond_to do |format|
-      format.json { render json: current_user.all_hooks }
-      format.yaml { render yaml: current_user.all_hooks }
+      format.json { render json: current_user.web_hooks, serializer: WebHookSerializer }
+      format.yaml { render yaml: JSON.parse(WebHookSerializer.new(current_user.web_hooks).to_json) }
```

Comment:
It is `JSON.parse(serialize_this.to_json)`. All yaml responses are [generated by parsing objects  serialized to json](https://github.com/rubygems/rubygems.org/pull/1409/files#diff-1a40f3055bea3743de857ed5e883f5c2R5).
`web_hooks` objects are serialized in `render` line itself because of the situation I have explained [here](https://github.com/rubygems/rubygems.org/pull/1409#discussion_r77448047).